models/intel/text-spotting-0004/description/text-spotting-0004.md

```diff
@@ -0,0 +1,89 @@
+# text-spotting-0004 (composite)
+
+## Use Case and High-Level Description
+
+This is a text spotting composite model that simultaneously detects and
+recognizes text. The model detects symbol sequences separated by space and performs
+recognition without a dictionary. The model is built on top of the Mask-RCNN
+framework with additional attention-based text recognition head.
+
+Alphabet is alphanumeric: `abcdefghijklmnopqrstuvwxyz0123456789`.
+
+## Example
+
+![](./text-spotting-0004.png)
+
+## Composite model specification
+
+| Metric                                        | Value     |
+|-----------------------------------------------|-----------|
+| Word spotting hmean ICDAR2015, without a dictionary | 68.40% |
+| Source framework                              | PyTorch\* |
+
+*Hmean Word spotting* is defined and measured according to the
+[Incidental Scene Text (ICDAR2015) challenge](https://rrc.cvc.uab.es/?ch=4&com=introduction).
+
+## Detector model specification
+
+The text-spotting-0004-detector model is a Mask-RCNN-based text detector with ResNet50 backbone and additional text features output.
+
+| Metric                                        | Value     |
+|-----------------------------------------------|-----------|
+| GFlops                                        | 184.495   |
+| MParams                                       | 27.010    |
+
+### Inputs
+
+1.	Name: `im_data` , shape: [1x3x768x1280]. An input image in the [1xCxHxW] format.
```

Comment:
We have agreement to use such format for input/output shape and layout specification (and plan to align it for all models), so please keep it:
```suggestion
1.	Name: `im_data` , shape: `1, 3, 768, 1280`. An input image in the `1, C, H, W` format.
```